go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	github.com/Azure/go-autorest/autorest v0.11.23
 	github.com/Azure/go-autorest/autorest/adal v0.9.18
 	github.com/TheThingsIndustries/mystique v0.0.0-20211230093812-d4088bd06959
-	github.com/TheThingsIndustries/protoc-gen-go-json v1.1.3
+	github.com/TheThingsIndustries/protoc-gen-go-json v1.2.1-0.20220119093152-ff4140a56565
```

Comment:
NOTE: This will be changed to a different version after https://github.com/TheThingsIndustries/protoc-gen-go-json/pull/5 gets merged.